remoteit/desktop
# [DESK-543][DESK-253][DESK-338] System tray launching fixes

### Description



- Added shared code directory to share resources between front and back end
- Refactored connection interface to support typeID 
- Renamed launchURL to LaunchTemplate
- Passed new launch application reference to tray so it could use the correct launch logic


### Checklist




### Test plan



1. Sign in
2. Connect to a service like vnc
3. Select the launch option in the system try menu
4. You should see it launch with the vnc:// path
5. Goto the connection details page
6. Edit the connection url
7. launch again with the system tray
8. You should see it launch with the edited url

### Ticket



<https://remoteit.atlassian.net/browse/DESK-543>
<https://remoteit.atlassian.net/browse/DESK-253>
<https://remoteit.atlassian.net/browse/DESK-338>
